resources/styles/components/scores/index.less

```diff
@@ -139,4 +140,21 @@
     }
   }
 
+  // concept coach cells that are split
+  .cc-cell, .name-cell {
```

Comment:
Could you put separate selectors in separate lines? That way it doesn't look like `.cc-cell .name-cell` :smile: